supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -27,6 +28,15 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
 
   private implicit val ec = context.dispatcher
   private val maxRetries = 10
+  // FIXME: Determined empirically that we start to get errors when the batch request approaches 15MB.
```

Comment:
Ok, now that we have more clarity, can you change this from a FIXME and give a little more context on what we found out so we remember the story if/when we come back around to this? Maybe a pointer to that one github link w/ the supposed fix and an explanation that this is in the "appears to work" category + your "it's actually length of time" theory?